app/views/schemesOverview.scala.html

```diff
@@ -74,7 +82,14 @@ <h2>@Html(Message("messages__schemesOverview__scheme_name", name))</h2>
         }
     }
 
-    <h2>@messages("messages__schemesOverview__UR__head")</h2>
-    <p>@messages("messages__schemesOverview__UR__text")</p>
-    <p><a id="user-research" href="@appConfig.userResearchUrl">@messages("messages__schemesOverview__UR__link")</a></p>
+    @variationSchemeName.map{ sn =>
+        <h2>@Html(Message("messages__schemesOverview__change_details__title", sn))</h2>
+        @variationDeleteDate.map{ dd =>
+            <p>@Html(Message("messages__schemesOverview__change_details__p1", sn, dd))</p>
+        }
+        <p><a id="continue-variation" href="">@messages("messages__schemesOverview__change_details_continue__link", sn)</a></p>
```

Comment:
links are missing for continue and delete

Reply:
There's a separate ticket to put the link targets in - in next sprint